lib/tasks/auto_annotate_models.rake

```diff
@@ -0,0 +1,59 @@
+# NOTE: only doing this in development as some production environments (Heroku)
+# NOTE: are sensitive to local FS writes, and besides -- it's just not proper
+# NOTE: to have a dev-mode tool do its thing in production.
+if Rails.env.development?
```

Comment:
I'm undecided on if I think this gem is worth it, but I think *if* we want to do it, we need to add this to the linter to help ensure it always stays up to date:

related: https://github.com/18F/identity-idp/blob/2755c7b54301e7b9eefd87911a6099723759f361/Makefile#L100-L101

Reply:
The original pull request comment from @n1zyy mentions it being run automatically on `db:migrate`, which could help keep it rather consistent? I'm not sure I follow how that's actually happening in the code currently, however.

>A previous project I was on used [the annotate gem](https://github.com/ctran/annotate_models), and it can be trivially set (as it is here) to run whenever `db:migrate` is run, so it will stay up to date.

Reply:
> I'm not sure I follow how that's actually happening in the code currently, however.

I was afraid someone would ask about this. It appears to work but I also have no idea how. 😇

Reply:
Looks like calling `Annotate.load_tasks` at the bottom of this file, which is outside the task block, will load tasks that override `db:migrate`:

https://github.com/ctran/annotate_models/blob/develop/lib/tasks/annotate_models_migrate.rake
And I think calling any rake task will initialize/load all the `lib/tasks` in this repo so that’s how it gets hooked up